app-conf/HeuristicConf.xml

```diff
@@ -193,5 +193,11 @@
     <classname>com.linkedin.drelephant.spark.heuristics.StagesHeuristic</classname>
     <viewname>views.html.help.spark.helpStagesHeuristic</viewname>
   </heuristic>
+  <heuristic>
+    <applicationtype>spark</applicationtype>
+    <heuristicname>Spark GC Time to Run Time</heuristicname>
```

Comment:
Would prefer naming this Heuristic as simply `Executor GC`.

Also reflect this change elsewhere in the Heuristic class and Help page.

Reply:
Done